csrc/host_ir/executor.h

```diff
@@ -139,7 +139,38 @@ class HostIrEvaluator final : public OptOutDispatch {
 
   c10::cuda::CUDAStream getCUDAStream(Stream* stream);
 
-  KernelArgumentHolder dispatchAndCollectOutputs();
+  Val* getAlias(Val* val) const {
+    const auto& aliases = container_->alias();
+    auto it = aliases.find(val);
+    return it != aliases.end() ? getAlias(it->second) : val;
+  }
+
+  bool isKnown(Val* value) const {
+    return expr_evaluator_.isKnown(getAlias(value));
+  }
+
+  PolymorphicValue getKnownConcreteData(Val* val) const {
```

Comment:
```suggestion
  PolymorphicValue getKnownConcreteValue(Val* val) const {
```
just to be consistent with `concrete_value` used in other places in this file.